src/formatter/rerun_formatter_spec.js

```diff
@@ -88,23 +91,36 @@ describe('RerunFormatter', function() {
     })
   })
 
-  describe('with two failing scenarios in different files', function() {
-    beforeEach(function() {
-      this.eventBroadcaster.emit('test-case-finished', {
-        sourceLocation: { uri: this.feature1Path, line: 1 },
-        result: { status: Status.FAILED }
-      })
-      this.eventBroadcaster.emit('test-case-finished', {
-        sourceLocation: { uri: this.feature2Path, line: 2 },
-        result: { status: Status.FAILED }
-      })
-      this.eventBroadcaster.emit('test-run-finished')
-    })
+  _.each(
+    [
+      { separator: null, label: 'default' },
```

Comment:
For this case, if I log the output it is `features/a.feature:1nullfeatures/b.feature:2`.  I believe it should be switched to undefined in order to trigger the default case.

Reply:
Nice catch. Actually I needed to define both the option and the expected separator.